src/screens/Gurbani/BottomBar.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
```

Comment:
We can move this to `src/components` and this will replace `src/components/ShortcutDrawer` ?

Reply:
Not for now. We will only move it if we require this component to be on other screens. I'm also inclined to just remove `ShortcutDrawer`, but let's press ahead with that later, once we're sure that `BottomBar` is functioning correctly.

Reply:
Right now, this component will only ever be on the `Gurbani` screen, according to the mocks.

Reply:
sounds good 👍🏼